src/pl/plpython/plpy_main.c

```diff
@@ -84,19 +84,19 @@ void PLy_handle_cancel_interrupt(void);
 bool PLy_enter_python_intepreter = false;
 bool PLy_path_added = false;
 
+static bool inited = false;
+
 /* GUC variables */
 #if PY_MAJOR_VERSION >= 3
-char *plpython3_path = NULL;
+static char *plpython3_path = NULL;
 
 bool 
 plpython3_check_python_path(char **newval, void **extra, GucSource source) {
-	if (PLy_path_added)
+	if (inited)
 	{
-		GUC_check_errmsg("SET PYTHONPATH for plpython3 can only set once in one session");
+		GUC_check_errmsg("SET PYTHONPATH faild, we can only set it before plpython3u init in one session");
```

Comment:
typo, `faild`, also maybe rephrase it to:

`The GUC value can only be changed before initializing the python interpreter.`